### ISSUE TITLE
bug: scrollAssist don't work at Page moved by navigateForward

### ISSUE THREAD

**Issue by rdlabo** [ionic-team/ionic#19472]
# Bug Report

**Ionic version:**
[x] **4.9.0 - 4.9.1**

__4.8.1 is worked__

**Current behavior:**
scrollAssist don't work at Page moved by navigateForward. ( worked in root page )

This is movie.
https://i.gyazo.com/f4ddb7facfe1412b81661e2c90926d55.gif

**Expected behavior:**
scrollAssist work at Page moved by push.

**Steps to reproduce:**
1. enable scrollAssist.

```
  imports: [
    IonicModule.forRoot({
      scrollAssist: true,
    }),
```

2. Put input area at Page moved by push.

**Related code:**

https://github.com/rdlabo/ionic-not-work-scrollAssist/commit/e32ac9c4cbb6f9dea861a2d7db929f83100dff96

**Other information:**
Maybe related this change:
https://github.com/ionic-team/ionic/commit/c3044f59df96666bbe005f3615300a5130ca8727

in @ionic/angular@4.8.1,  scrollAssist is worked.

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 5.4.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.9.1
   @angular-devkit/build-angular : 0.801.3
   @angular-devkit/schematics    : 8.1.3
   @angular/cli                  : 8.1.3
   @ionic/angular-toolkit        : 2.0.0

Capacitor:

   Capacitor CLI   : 1.2.1
   @capacitor/core : 1.2.1

Utility:

   cordova-res : 0.6.0 
   native-run  : not installed

System:

   NodeJS : v10.16.2 (/usr/local/bin/node)
   npm    : 6.10.3
   OS     : macOS Mojave
```

**Comment by capc0** [comment]
any updates yet?

**Comment by rdlabo** [comment]
@mhartington I tried to fix it, but I couldn't... If possible, could you work on it?🙏

**Comment by liamdebeasi** [comment]
Thanks for the issue! There have been numerous keyboard improvements that have shipped as of Ionic Framework v5.1.1. I strongly encourage you to update your applications and try the improvements as they should resolve this issue. 

Please open a new issue for any new bugs you run into. Thank you!
